src/state/slices/opportunitiesSlice/selectors/combined.ts

```diff
@@ -241,8 +241,44 @@ export const selectAggregatedEarnOpportunitiesByAssetId = createDeepEqualOutputS
 
     const sortedOpportunitiesByFiatAmountAndApy = activeOpportunities.concat(inactiveOpportunities)
 
+    const filterThorchainSaversZeroBalance = (
+      opportunities: AggregatedOpportunitiesByAssetIdReturn[],
+    ) => {
+      return (
+        opportunities
+          .map(opportunity => {
+            // Individual opportunities - all but savers
+            const filteredStakingOpportunities = opportunity.opportunities.staking.filter(
+              opportunityId => {
+                const maybeOpportunity = combined.find(opp => opp.id === opportunityId)
+                if (!maybeOpportunity) return true
```

Comment:
Why do we keep the opportunity in the list if it's falsey?

```suggestion
                if (!maybeOpportunity) return false
```

Reply:
🧠  6c41aa4142 / https://jam.dev/c/e870bb4d-923f-4aad-8821-89e56fa0e1dc